docs/guides/how_to_guides/creating_and_editing_expectations/how_to_create_custom_expectations.rst

```diff
@@ -646,16 +648,17 @@ Beginning in version 0.13, we have introduced a new API focused on enabling Modu
         7. **Import**: To use a custom Expectation, you need to ensure it has been imported into the running python interpreter. While including the module in your `plugins/` directory will make it *available* to import, you must still import the Expectation:
         
         .. code-block:: python
-            # get a validator 
-            # Note: attempting to run our expectation now would fail, because even though 
-            # our Expectation is in our DataContext plugins/ directory it has not been imported.
-            
-            from custom_module import ExpectColumnMaxToBeBetweenCustom
-            
-            # now we can run our expectation
-            validator.expect_column_max_to_be_between_custom('col', min_value=0, max_value=5)
-
-        7. **Optional:** Implement :ref:`Custom Data Docs Renderers <how_to_guides__configuring_data_docs__how_to_create_renderers_for_custom_expectations>`
+
+           # get a validator
```

Comment:
I noticed this section doesn't render correctly, and I hope this indentation change fixes it, but I don't really know enough about rst to know for sure.

This section [on main (without this PR)](https://docs.greatexpectations.io/en/latest/guides/how_to_guides/creating_and_editing_expectations/how_to_create_custom_expectations.html) currently renders like this for me:

![image](https://user-images.githubusercontent.com/3752732/120565584-a4c33a00-c3ca-11eb-963b-45a1fd2f5ab5.png)